Code/src/main/java/nl/utwente/group10/ui/components/FunctionBlock.java

```diff
@@ -21,34 +17,31 @@
 public class FunctionBlock extends Block {
 	/** The arguments this FunctionBlock holds.**/
 	private String[] arguments;
-	/** The name of this Function.**/
-	private String functionName;
-	
-	/**
-	 * Method that creates a newInstance of this class along with it's visual representation
-	 * @param the number of arguments this FunctionBlock can hold
-	 * @return a new instance of this class
-	 * @throws IOException
-	 */
-	public static FunctionBlock newInstance(int numberOfArguments) throws IOException {
-		FunctionBlock functionBlock = (FunctionBlock) FXMLLoader.load(Main.class.getResource("/ui/FunctionBlock.fxml"), null, new TactileBuilderFactory());
-		functionBlock.initializeArguments(numberOfArguments);
 
-		return functionBlock;
+	/** The name of this Function. **/
+	private StringProperty name;
+
+	/** The type of this Function. **/
+	private StringProperty type;
+
+	@FXML
+	private Pane nestSpace;
```

Comment:
nestSpace will automatically be filled from whatever has a `fx:id` of nestSpace.